components/state/api_version.go

```diff
@@ -36,20 +36,26 @@ func UpdateAPIVersionIfNotExists(ctx context.Context, db *sql.DB, apiVersion str
 }
 
 func UpdateAPIVersion(ctx context.Context, db *sql.DB, apiVersion string) error {
-	query := fmt.Sprintf(`
-INSERT OR REPLACE INTO %s (%s) VALUES (?)
-`, TableNameAPIVersion, ColumnAPIVersion)
+	// First delete any existing version
+	query := fmt.Sprintf(`DELETE FROM %s`, TableNameAPIVersion)
```

Comment:
what is the issue with `insert or replace`?

Reply:
there was some bug when reading this value, fixed the other way, PTAL.